.github/workflows/ci-testing-deploy.yml

```diff
@@ -912,7 +912,10 @@ jobs:
         run: ./ci/helpers/show_system_versions.bash
       - name: install
         run: ./ci/github/unit-testing/postgres-database.bash install
+      - name: typecheck
+        run: ./ci/github/unit-testing/postgres-database.bash typecheck
       - name: test
+        if: always()
```

Comment:
what is the purpose of this `if always()`, does removing it not do the same thing?

Reply:
This makes sure both ``checkstyle`` and ``test`` are always run (even if one of the fails).